packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -181,59 +199,24 @@ export default class SaytDriverPlugin implements Plugin {
 
   /**
    * Extracts query and products from the given response.
+   * Calls `this.transformProduct` on each product found in the response.
```

Comment:
You can link directly to the property.

```suggestion
   * Calls [[transformProduct]] on each product found in the response.
```